test/Libraries/RevitIntegrationTests/ElementTests.cs

```diff
@@ -37,6 +37,33 @@ public void CanDeleteElement()
             // query count node to verify 1 item deleted as a result of the wall deletion. 
             Assert.AreEqual(1, GetPreviewValue("ccd8a5ba37fd4b1297def564392ccf54"));
          }
-        
+
+        [Test]
+        [TestModel(@".\element.rvt")]
+        public void CanGetPinnedStatus()
+        {
+            string samplePath = Path.Combine(workingDirectory, @".\Element\deleteWallFromDocument.dyn");
+            string testPath = Path.GetFullPath(samplePath);
+
+            ViewModel.OpenCommand.Execute(testPath);
+
+            RunCurrentModel();
+        }
+
+        /// <summary>
+        /// Checks if Elements pin status can be set from Dynamo
+        /// </summary>
+        [Test]
+        [TestModel(@".\element.rvt")]
+        public void CanSetPinnedStatus()
+        {
+            string samplePath = Path.Combine(workingDirectory, @".\Element\setElementsPinStatus.dyn");
+            string testPath = Path.GetFullPath(samplePath);
+
+            ViewModel.OpenCommand.Execute(testPath);
+
+            RunCurrentModel();
```

Comment:
we need to

1. get element pinned status before we ran the graph
2. get element pinned status after running
3. Assert the action changed the pinned status as expected